src/common/Popups.js

```diff
@@ -0,0 +1,207 @@
+                  variant="outlined" 
+                  size="small" 
+                  style={{width: "300px" }}
+                  />
+              </FormControl>
+            </FormGroup>
+          </div>
+        </div>
+      </Form>
+    );
+    const updateUsername = (
+      <Form>
+        <div className="row">
+          <div className="col-md-12">
+             <FormGroup >
+              <FormControl>
+                <TextField 
+                  label="Username" 
+                  placeholder="Username" 
+                  defaultValue={optionValue} 
+                  name="username" 
+                  onChange={this.onChange} 
+                  variant="outlined" 
+                  size="small" 
+                  style={{width: "300px" }}/>
+              </FormControl>
+            </FormGroup>
+          </div>
+        </div>
+      </Form>
+    );
+    const deactivateAccount = (
+      <div className="container">
+        <p className="text text-center" style={{color: "red"}}><b>Are you sure ??</b></p>
+        <div className="row">
+          <div className="col-md-2"></div>
+            <Button 
+              className="col-md-3" 
+              variant="outlined" 
+              color="primary" 
+              onClick={this.handleClose}
+              >No
+              </Button>
+          <div className="col-md-2"></div>
+            <Button 
+              className="col-md-3"
+              color="secondary"
+              variant="outlined"
+              onClick={this.handleDeactivateAccount}
+              >Yes
+              </Button>
+          <div className="col-md-2"></div>
+        </div>
+      </div>
+    );
+
+    const checkVerification = (
+      <div>
+        <div className="row">
+          <div className="col-md-12">
+            <div className="center">{ optionValue ? (<div>
+               <center>
+                <img 
+                  className="img-fluid img-circle"
+                  src={identityImage} 
+                  alt="identity badge" 
+                  height="150px"
+                  width="150px"
+                />  
+                <p className="text lead"><b>Verified, Thanks</b></p>
+                </center>            
+            </div>) : 
+            ("Not verified")}</div>
+          </div>
+        </div>
+      </div>
+    )
+    return (
+       <Modal show={this.state.show} onHide={this.handleClose} animation={false} centered>
+        <Modal.Header closeButton>
+          <Modal.Title>Update { " " + option }</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          {Boolean(option === "name") ? updateName : null}
+          {Boolean(option === "email") ? updateEmail: null}
+          {Boolean(option === "username") ? updateUsername: null}
+          {Boolean(option === "account") ? deactivateAccount: null}
+          {Boolean(option === "identity") ? checkVerification : null}
+        </Modal.Body>
+        <Modal.Footer>
```

Comment:
use for loop store keys in an array

Reply:
@vaibhavdaren I have not done these changes, these changes are there as I have just fetched the development branch code and started working on that code.
So should I resolve this here or in development branch.

Reply:
The problem was that the Material-UI branch does not have an updated code of development branch

Reply:
@vaibhavdaren

Reply:
Ok.

Reply:
Start of here slowly we will move to development branch

Reply:
cool, I will do the required changes here!

Reply:
done!